integrated_channels/integrated_channel/exporters/content_metadata.py

```diff
@@ -192,7 +282,7 @@ def _get_catalog_diff(self, enterprise_catalog, content_keys, force_retrieve_all
             enterprise_catalog,
             items_to_delete
         )
-        return items_to_create, content_to_update, content_to_delete
+        return content_to_create, content_to_update, content_to_delete
```

Comment:
pass a dict of content_key : transmission item, consistent with the other payloads